### ISSUE TITLE
Bug: OpenPype 3.15.11 / WWG III / PUBLISH KITSU / SHOT40 / ANIM COLO TASK 

### ISSUE THREAD

**Issue by Alice51000** [quadproduction/OpenPype#510]
### Before posting a new ticket, have you looked through the documentation to find an answer?

Yes I have

### Have you looked through the existing tickets to find any related issues ?

Yes I have

### Author of the bug

Alice Sammut

### Version

3.15.11

### What platform you are running OpenPype on?

Windows

### Current Behavior:

Hello

we're facing several issues when trying to publish preview in kitsu with the updated version of OpenPype.

We can't uncheck the mark in and out like we used to with the previous version. Our publish doesn't pass studio validation because we often need more or less frames in the animation step to work. Do you think it something we can change for this version ?
![image](https://github.com/quadproduction/OpenPype/assets/134387402/b57e6546-842b-4b2b-82b6-8414880a4f2a)

If we over pass this issue for the publish and handle the in and out in the file, it passes the studio validation, the publish is validate BUT it doesn't appear in kitsu. Can you have a look ?

FYI : Hadrien our TD is working on the Adolescence project and is in holidays starting tomorrow

### Expected Behavior:

I am expecting a publish in kitsu

### What type of bug is it ?

Happened only once in a particular configuration

### Which project / workfile / asset / ...

Wizard With a Gun III / ANIMATION / COLO / SHOT 40

### Steps To Reproduce:

1. Launch publishing
2. Step mark in and out failed
3. Precised the in and out by hand
4. Publish succeed
5. No publish in kitsu

### Additional context:

We are currently testing a new versioning of OpenPype

**Comment by hfarre** [comment]
Concerning the Mark In Mark Out, it's there now:


![Image](https://github.com/quadproduction/OpenPype/assets/139864808/ff867eef-7c19-4e8c-acf5-fe36902471e7)

**Comment by hfarre** [comment]
Hello
Is someone working on it at the moment ?

**Comment by ccaillot** [comment]
I replace this parameter "Add Review to FTrack" by "Add Review to Kitsu" in "Project settings > photoshop > Publish > Extract Review" in WIZARDS_WITH_GUNS_3, Can you tell me if this change something on your side ?

**Comment by ccaillot** [comment]
On it
